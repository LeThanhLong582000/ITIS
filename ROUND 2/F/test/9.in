100
420532 725495 896054
488930 624137 400134
646676 588172 31663
468398 959818 659484
579376 447704 120120
709763 330634 799123
340986 488570 789440
92346 645754 30693
274883 344993 185239
150359 617853 554051
149822 222518 578860
887181 532426 321324
307543 113298 667216
475009 342605 790210
51322 837454 279587
272484 409544 102046
39233 244247 692945
873243 769626 287219
613121 634365 752047
594538 923192 491881
347353 255203 23301
478694 277726 896179
854043 416699 744479
922728 163440 709051
819030 511386 809014
247289 63248 176274
801695 552764 991420
811281 610707 651083
982469 269373 106416
929535 389930 735664
747423 9918 509390
745926 241030 418776
155574 719038 989719
402847 314553 464560
52548 275583 677429
868083 377377 759022
59885 816627 331371
239452 622036 539892
416263 662311 643884
429886 72422 715295
727688 528330 651982
921990 472598 683373
169856 566898 236967
149649 255452 712566
906781 104296 698610
732148 684646 488392
645818 177803 873644
415623 682163 26005
406727 549761 481714
45708 705388 126706
873321 359539 327584
557418 197075 381583
67807 291170 822223
181752 460818 798226
368338 726066 772647
670878 476892 37094
560497 94327 565640
119727 176004 50290
218405 334838 552265
657929 307282 903554
561871 49380 411566
329462 88449 249417
155013 649563 685136
506230 999926 531589
386991 587353 418060
667426 122238 623224
802477 886236 878694
425705 393799 579117
72384 870523 688815
945509 722365 607102
293407 125000 911188
972983 522848 689446
358118 547367 220861
190932 952212 439103
990853 930530 906251
39540 814997 210287
761570 851400 529242
446531 688325 120741
304726 365916 405684
361649 542363 276939
543115 852509 997149
619764 19741 745817
563318 731610 821162
728779 186995 93223
452945 636482 173761
694437 680712 562821
475048 415277 846544
503692 143394 943031
323575 346761 259666
116329 937747 915802
613768 863781 614709
22768 102041 716702
471212 815910 306047
68013 234886 679261
235755 171153 58725
741045 449007 619645
621589 367631 991638
563216 231322 384508
996030 622826 902944
684618 167386 594456
