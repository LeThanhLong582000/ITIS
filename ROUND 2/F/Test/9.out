295546
132464
31664
84032
120121
589126
320928
0
57565
43279
253082
321325
430731
461307
0
0
243966
195418
365402
81614
23302
548574
590912
709052
558329
176275
620176
425829
106417
637635
509391
418777
213128
276427
227197
624864
0
78654
198918
536380
425670
566383
0
303382
698611
267947
670830
0
169340
0
327585
370963
299430
259580
207460
37095
515905
0
217916
627101
411567
245215
95293
18947
108849
584206
397468
305512
0
415123
539798
569791
15806
0
483287
0
219706
0
172247
48113
343878
672920
326435
93224
0
288273
453158
651665
118240
47192
182348
318715
0
256194
58726
455842
622798
358201
638074
555844
